### PR TITLE
Minor cleanup after Eric's feedback.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/src/rdm_responder.c [hunk 1]
```diff
@@ -412,7 +412,12 @@ int RDMResponder_GetDeviceInfo(const RDMHeader *header,
   ptr = PushUInt32(ptr, g_responder.def->software_version);
   ptr = PushUInt16(ptr, personality ? personality->dmx_footprint : 0);
   *ptr++ = g_responder.current_personality;
-  *ptr++ = g_responder.def->personality_count;
+
+  if (g_responder.def->personalities) {
+    *ptr++ = g_responder.def->personality_count;
+  } else {
+    *ptr++ = 1;
```
Comment (peternewman) [inline firmware/src/rdm_responder.c]: Really? If a device has no footprint, how can it have a personality?

**Comment by peternewman** [comment]
Generally LGTM

File: firmware/src/rdm_responder.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/rdm_responder.c]: It's open to interpretation. From talking to people it seems 1 / 1 is the unofficial interpretation. I've added it to the fix it list for E1.20.

Comment (peternewman) [inline firmware/src/rdm_responder.c]: Okay

**Pull request merged**